tests/unit/components/ItemLookup.test.ts

```diff
@@ -142,7 +142,98 @@ describe( 'ItemLookup', () => {
 				},
 			] );
 		} );
+
+		it( ':itemSuggestions - shows matching suggestions before search results', async () => {
```

Comment:
This test should be part of the `:props` describe blog above

Reply:
It is part of the block, isn’t it?
![image](https://user-images.githubusercontent.com/2346599/167436210-57a5c1cf-2e59-45bf-adb9-d3c250e55527.png)

Reply:
You're right! It seems GitHub is using different indents for code that is shown initially in a diff and code that is initially folded away and only revealed on user click. That confused me because the `it(` calls didn't line up.

Reply:
Right, that’s annoyed me before… I think for initially shown code it respects your tab width preference (4 characters for me), but for extra context it always seems to use 8 characters?